yarn.lock

```diff
@@ -0,0 +1,99 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+asap@~2.0.3:
```

Comment:
remove this file please?

Reply:
Disagree about removing this one. I use yarn, it's gaining popularity, I think it's better than npm and it's a drop-in replacement. Doesn't hurt to have the file, can be ignored if not using.